AutomatedTesting/Gem/PythonTests/editor/EditorScripts/Component_PhysX_Dynamic Rigid_Body.py

```diff
@@ -0,0 +1,73 @@
+"""
```

Comment:
This file name has a space in it.

`Component_PhysX_Dynamic Rigid_Body.py`

Reply:
added an underscore to fix this filename